Collate v6.4.0 release notes

sf1919 opened the pull request:
**Description of work.**

This PR collates recent release notes that have been added since initial cleaning up of the bulk of the release notes. Please note:- files will have headings with no notes as the plan is the remove these in the final cleaup of the notes.

**To test:**

- remove `build/docs/doctrees`
- build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Grammar/Spelling check

*There is no associated issue.*


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

sf1919 committed:
Add extra v6.4.0 Framework release notes

- Added new release notes since last tidy of v6.4.0 Framework release notes

sf1919 committed:
Added extra v6.4.0 Diffraction release notes

- Added extra release notes added since first cleanup of v6.4.0 Diffraction release notes

sf1919 committed:
Added new v6.4.0 Direct Geometry release notes

- added release notes added since first tidy of v6.4.0 Direct Geometry Release notes

sf1919 committed:
Added extra v6.4.0 Indirect release notes

- Added release notes since Indirect v6.4.0 release notes initially tidied.

sf1919 committed:
Added new SANS release note for v.6.4.0

sf1919 committed:
Added new release notes to v.6.4.0 Workbench release notes

Harrietbrown commented:
At the top of my documents I have this:
![image](https://user-images.githubusercontent.com/49688535/172350888-517aa091-8683-4a05-83c8-c753f766d9e0.png)
It is in all my release docs.

sf1919 committed:
Remove MSlice release note

- A new feature is no longer going forward in MSlice for this release. Therefore information about it needs to be removed from the release notes.

Harrietbrown commented:
Looking over the notes they all look good to me. If a second build shows that the issue I was happening is only happening on my build then this is good to go.

Harrietbrown reviewed (approved):
The angstrom issue seems to have been excllusive to my build, in which case the notes look good. Good Job.
